wls-gui-wahllokalsystem/src/components/navigation/auszaehlung_wahlarten/TheMBWScoresListGroup.vue

```diff
@@ -53,6 +54,7 @@ const listItems = [
   { title: "Ungültige Stimmzettel", routeName: ROUTE_STAPEL_D },
   { title: "Gültige Stimmzettel", routeName: ROUTE_STAPEL_A_AND_B },
   { title: "Schnellmeldung", routeName: ROUTE_SCHNELLMELDUNG },
+  { title: "Kandidatinnenstimmen", routeName: ROUTE_STAPEL_BC },
```

Comment:
```suggestion
  { title: "KandidatInnenstimmen", routeName: ROUTE_STAPEL_BC },
```
gibts da vorschriften wegen dem gendern? also ob man zB `*` oder `/` benutzt oder so wie ich vorgeschlagen habe?

Reply:
Habe mich jetzt für den alten Titel entschieden: `Kandidatinnen- und Kandidatenstimmen` (c1d9ef6212b167b6ac04bc8b1999938fdb0e3ff6)